low-level/files/opt/seagate/sspl/setup/setup_logger.py

```diff
@@ -27,11 +29,17 @@ def init_logging(syslog_host="localhost", syslog_port=514, console_output=False)
     """Initialize logging for sspl-setup."""
     _logger.setLevel(logging.INFO)
     # set Logging Handlers
-    handler = logging.handlers.SysLogHandler(
-                address=(syslog_host, syslog_port))
-    syslog_format = "%(name)s[%(process)d]: " \
-                    "%(levelname)s %(message)s (%(filename)s:%(lineno)d)"
-    formatter = logging.Formatter(syslog_format)
+    Utility().create_file(SETUP_LOG_PATH)
+
+    handler = logging.handlers.RotatingFileHandler(
```

Comment:
Ok this is enabled for sspl-setup logs only, that way we are doing away with additional config file also. But we will loose flexibility to have config params for this, like maxBytes & backup count here. 

Can you put these configs in sspl.conf so we can use from there.

If this is seen stable, same way we can adopt for sspl & other logs also

Reply:
> Can you put these configs in sspl.conf so we can use from there.

 @sandeepsgt @cdeshmukh
Then we will land up in the same problem as earlier,
 as sspl.conf is being created & loaded by post-install process stage.
 and here init_logging function is being called by sspl-setup script, before post-install interface.

Reply:
can we keep these conf values in sspl_constant.py instead?

Reply:
I am fine if values are not configurable for this. Because I feel there we don't need to change this once code is deployed. Also, SSPL setup logs will be very small in amount. So, current values should be fine
Just FYI - Earlier also it was [not configurable](https://github.com/Seagate/cortx-monitor/blob/main/low-level/files/etc/logrotate.d/sspl_setup_logs)

Reply:
Ok, i think let this be, given these are setup logs only. how often setup would be executed on a deployed system anyways. Current number suffice, including rotation